lua/wikis/commons/MatchGroup/Legacy.lua

```diff
@@ -422,4 +423,127 @@ function MatchGroupLegacy:build()
 	return MatchGroup.Bracket(self.newArgs)
 end
 
+--does the same as run but returns wiki code instead of displayed bracket
+---@return string
+function MatchGroupLegacy:generate()
+	local args = self.args
+
+	self.bracketType = args.type
+	local match2mapping = self:_get(args.template, args.templateOld)
+
+	self.newArgs = {
+		args.template,
+		id = args.id,
+		store = args.store,
```

Comment:
```suggestion
		storeMatch2 = args.store,
```
Isn't this correct?

Reply:
store is correct as it affects both match1 and match2 storage

https://github.com/Liquipedia/Lua-Modules/blob/cb2f16f13424e2583b376117ab604c851f4ecf5f/lua/wikis/commons/MatchGroup/Base.lua#L31-L40